index.html

```diff
@@ -11,6 +11,1053 @@
+                    <p>By <a href="#">JOSHUA C. WILSON</a></p>
+                  </div>
+                </div>
+
+                <div class="py-3 opinion-box d-flex flex-row">
+                  <img src="https://d.newsweek.com/en/full/1598216/joshua-c-wilson.png?w=63&h=63&f=10000e40757c6afd0e47058d1e010205" alt="">
+                  <div class="content">
+                    <h6><a href="#">Can Biden Unify the Country? California v. Texas Says It All</a></h6>
+                    <p>By <a href="#">JOSHUA C. WILSON</a></p>
+                  </div>
+                </div>
+
+                <div class="py-3 opinion-box d-flex flex-row">
+                  <img src="https://d.newsweek.com/en/full/1598216/joshua-c-wilson.png?w=63&h=63&f=10000e40757c6afd0e47058d1e010205" alt="">
+                  <div class="content">
+                    <h6><a href="#">Can Biden Unify the Country? California v. Texas Says It All</a></h6>
+                    <p>By <a href="#">JOSHUA C. WILSON</a></p>
+                  </div>
+                </div>
+
+              <div class="mid-card">
+                <div class="card" style="width: 18rem;">
+                  <img class="card-img-top" src="https://d.newsweek.com/en/full/1657994/cul-map-otherworldly-banner.webp?w=397&h=397&f=47c67cfe599f41a62cc265fbfd773556" alt="Card image cap">
+                  <div class="card-body">
+                    <h5 class="card-title"><a href="#"> Finland: Sustainability and Innovation</a></h5>
+                    <p class="card-text">The country is powering through a pandemic that brought global growth to a standstill, supported by its world-leading tech sector and innovative industries.</p>
+                  </div>
+                </div>
+
+                <form method="post">
+                  <div class="d-flex flex-row">
+                    <img src="https://g.newsweek.com/www/images/logo-n1.svg" alt="">
+                    <h4>START YOUR DAY WITH OUR TOP 5 ARTICLES</h4>
+                  </div>
+                  <input class="my-2" type="email" name="email" placeholder="Email address" required>
+                  <button type="submit">FREE SIGN UP ></button>
+                </form>
+                
+                <div class="card" style="width: 18rem;">
+                  <img class="card-img-top" src="https://d.newsweek.com/en/full/1657994/cul-map-otherworldly-banner.webp?w=397&h=397&f=47c67cfe599f41a62cc265fbfd773556" alt="Card image cap">
+                  <div class="card-body">
+                    <h5 class="card-title"><a href="#"> Finland: Sustainability and Innovation</a></h5>
+                    <p class="card-text">The country is powering through a pandemic that brought global growth to a standstill, supported by its world-leading tech sector and innovative industries.</p>
+                  </div>
+                </div>
+
+                <div class="card" style="width: 18rem;">
```

Comment:
- [ ] Remove this inline styling, this is one of the things affecting the horizontal scroll on your page.